src/fiuba/algo3/tp2/PicoPiedra.java

```diff
@@ -0,0 +1,10 @@
+package fiuba.algo3.tp2;
+
+import fiuba.algo3.tp2.estrategiasDesgaste.EstrategiaDesgasteDividida;
+
+public class PicoPiedra extends Pico {
+    public PicoPiedra() {
+        this.fuerza = 4;
+        this.desgastador = new EstrategiaDesgasteDividida(200, this.fuerza, 1.5);
```

Comment:
Lo mismo que arriba, se podria pasar por parametro this.durabilidad en vez del 200?